src/OauthReceiver/index.js

```diff
@@ -57,7 +57,7 @@ class OauthReceiver extends React.Component {
         ...args,
       });
 
-      const headers = new Headers({ 'Content-Type': 'application/json' });
+      const headers = new Headers({ 'Content-Type': 'application/x-www-form-urlencoded' });
```

Comment:
this it not "really" useful as it can be overwritten indeed.
This was more in a logic to follow rfc guidance: https://tools.ietf.org/html/rfc6749  (top of p17) 
as suggest in the original depository : https://github.com/adambrgmn/react-oauth-flow/pull/40

Reply:
👍